hazelcast/src/main/java/com/hazelcast/map/impl/operation/steps/IMapStepAwareOperation.java

```diff
@@ -42,6 +42,6 @@ default Step getStartingStep() {
             return UtilSteps.DIRECT_RUN_STEP;
         }
 
-        return StepAwareOperation.super.getStartingStep();
+        return UtilSteps.DIRECT_RUN_STEP;
```

Comment:
now that this method can only return DIRECT_RUN_STEP, what about removing the if statement?

Reply:
current use-case only requires to offload tstore backup operations, no need to offload other backup ops in cases such that map-store is enabled.